java/test/jmri/jmrit/beantable/TurnoutTableActionTest.java

```diff
@@ -55,6 +116,7 @@ public void setUp() {
     public void tearDown() {
         a = null;
         jmri.util.JUnitUtil.resetInstanceManager();
+        jmri.util.JUnitUtil.initDefaultUserMessagePreferences();
```

Comment:
When cleaning up after tests, you want to leave the system as unconfigured as possible, so setting the default user message preferences is not appropriate here.